app/concepts/clients/use_cases/find_by.rb

```diff
@@ -0,0 +1,15 @@
+module Clients
+  module UseCases
+    class FindBy
+      attr_reader :repository
+
+      def initialize(repository: Clients::Repository.new)
+        @repository = repository
+      end
+
+      def call(id:)
+        repository.find_by(id)
```

Comment:
Ok, so this use case is just calling `find_by` on repository - it is a bit too much abstraction in this case - calling directly repository instead of that use case would work as well.

Reply:
Yes. It was already pointed out in this discussion https://github.com/agusiol/montecinema/pull/14#discussion_r646317115
I have few of those usecases that are not needed so I will have to refactor it a bit.